README.md

```diff
@@ -148,6 +150,12 @@ Then, rerun the `make deploy` copmmand to update your stack's settings.
 
 With the Postgres container enabled, users (both students and instructors) can connect to a shared Postgres database from within their Jupyter Notebooks by opening a connection with the standard `psycop2g` depency using the `postgres-labs` host name. IllumiDesk's [user guides provide additional examples](https://docs.illumidesk.com) on the commands and common use-cases available for this option.
 
+### Shared Folder
+
+Users that wish to use a shared folder between Instructors and Learners may do so by setting the `shared_folder_enabled` to true.
+
+With the Shared folder enabled, users with access to the grader service (instructors/TA) can create files directly in their `/shared` folder. Since each grader service is launched by each course so all the files created here must be reflected within the `/shared/<course_name>` in all the learner containers.
```

Comment:
With `shared_folder_enabled` set to true, users with access to the shared grader service (by default Instructors and TAs) may create files directly in the course's `/shared` folder. Since one shared grader notebook is launched for each course then all the files created in the `/shared` folder appear within the `/shared/<course_name>` in all end-user workspaces.